gasprice/feehistory.go

```diff
@@ -0,0 +1,141 @@
+package gasprice
+
+import (
+	"encoding/binary"
+	"errors"
+	"log"
+	"math"
+	"math/big"
+	"sort"
+)
+
+var (
+	ErrInvalidPercentile = errors.New("invalid percentile")
+	ErrBlockCount        = errors.New("blockCount must be greater than 0")
+	ErrBlockInfo         = errors.New("could not find block info")
```

Comment:
To be more consistent with rest of the client:
 
```suggestion
	ErrBlockNotFound         = errors.New("could not find block")
```